build/ci/evergreen.yml

```diff
@@ -1036,6 +1036,7 @@ tasks:
             - src/github.com/mongodb/mongocli/dist/*.rpm
             - src/github.com/mongodb/mongocli/dist/*.tgz
             - src/github.com/mongodb/mongocli/dist/*.json
+            - src/github.com/mongodb/mongocli/dist/*.msi
```

Comment:
I have just noticed that we already put msi to the download center in `release_msi`

Reply:
to show in the download centre you also need to update https://github.com/mongodb/mongocli/blob/6d025f8e13fc69d3ac2de78363005fc6d750de2d/internal/release/main.go